tasks/verify-enterprise-contract.yaml

```diff
@@ -17,6 +17,15 @@ spec:
       type: string
       description: Image reference to verify
 
+    - name: PIPELINERUN_NAME
+      type: string
+      description: The name of the pipelinerun to gather task results from
```

Comment:
Thinking about the implications of demoing here - if the param is required it's harder to demo without a test task in place. Should we consider making the param optional, so we the existing demo works when there is no handy test output?

Or is it easy enough to prepare some hacbs test output for demo purposes?

Reply:
Not a blocker, but something to consider.

Reply:
This is a good question. Working with the test results was an integral part of the demo, so I didn't consider making it optional, but it does take away from the flexibility.

Reply:
Yeah, Zoran was thinking about how to address this with an updated demo script. (As per slack discussion.)